### DISCUSSION TITLE
Add ability to change unit of measures

### DISCUSSION THREAD

**Discussion by beeaton** [plmilord/Hass.io-custom-component-ikamand#2] [Ideas]
Can you look to add a way to choose which unit of measures to output?  I see the main temp is shown in Fahrenheit while the induvial probes show in Celsius.  Great job so far on this component!!!

**Comment by plmilord** [comment]
I'm definitely going to take a look! This should be a walk in the park as I have another integration that requires such treatment!

**Reply by plmilord** [reply]
iKamand v1.1 now fully supports changing from Celsius to Fahrenheit and vice versa!

**Reply by beeaton** [reply]
Awesome work!!!
